login-fire-form.html

```diff
@@ -296,7 +296,8 @@
     },
 
     listeners: {
-      'error': '_handleError'
+      'error': '_handleError',
+      'signedin': '_handleSignIn'
```

Comment:
What about call directly the `_cleanInputs` function and remove `_handleSignIn`?